docs/GetStarted/generic_vulkan_env_setup.md

```diff
@@ -0,0 +1,128 @@
+# Generic Vulkan Development Environment Setup and Troubleshooting
+
+[Vulkan](https://www.khronos.org/vulkan/) is a new generation graphics and
+compute API that provides high-efficiency, cross-platform access to modern GPUs
+used in a wide variety of devices from PCs and consoles to mobile phones and
+embedded platforms.
+
+This page lists steps and tips for setting up and trouble shooting a Vulkan
+development envirnoment. The information here is meant to be generic.
+
+## Vulkan architecture
+
+Vulkan adopts a layered architecture, which aims to better support extensiblity.
+There are four components involved in this architecture:
+
+* The Vulkan Application
+* [The Vulkan Loader][VulkanLoader]
+* [Vulkan Layers][VulkanLayer]
+* [Installable Client Drivers (ICDs)][VulkanICD]
+
+![High Level View of Loader][VulkanArchPicture]
+
+The Vulkan loader sits between the Vulkan application, which calls Vulkan APIs,
+and the ICDs, which implements these Vulkan APIs. Vulkan layers agument the
+Vulkan system to provide optional features like validation and debugging.
+The Vulkan loader composes a chain of requested layers, which processes the
+Vulkan application's API calls one by one and finallly redirects the API calls
+made by the Vulkan application to one or more ICDs.
+
+It's highly recommned to read the [Architecture of the Vulkan Loader Interfaces
+Overview][VulkanArchOverview] to get a general understanding of what these
+components are and how they interact with one another.
+
+## Vulkan development environment setup
+
+### Windows
+
+You need to install the [Vulkan SDK][VulkanSDK] from LunarG to get the Vulkan
+loader.
+
+Typically the Vulkan SDK will be installed at `C:\VulkanSDK\<version>\` and
+there will be an environment variable `VULKAN_SDK` pointing to it. You can run
+the `vulkancube` executable under the `Bin\` subdirectory of the Vulkan SDK to
+make sure everything works properly. If not, you probably need to check whether
+the graphics card is Vulkan capable or update the driver.
+
+### Debian/Ubuntu
+
+The following packages should be installed for a proper Vulkan runtime:
+
+* [libvulkan1][PackageLibVulkan1] for the Vulkan loader `libvulkan.so`.
+* (AMD) [mesa-vulkan-drivers][PackageMesaVulkan] for Mesa AMD Vulkan ICD.
+* (NVIDIA) [nvidia-vulkan-icd][PackageNvidiaVulkan] for NVIDIA Vulkan ICD.
+
+The above packages provide the Vulkan loader and ICDs. With them an Vulkan
+application should be able to run. You may additionally want to install
+
+* [vulkan-tools][PackageVulkanTools] for command-line tools like `vulkaninfo`
+  (dumping available ICDs and their capabilities) and GUI application like
+  `vulkancube` (rendering a rotating cube).
+
+To test the runtime functions properly.
```

Comment:
nit: sentence fragment, could lowercase 'to' so this continues the previous fragment?